src/garage/torch/value_functions/gaussian_mlp_value_function.py

```diff
@@ -0,0 +1,130 @@
+"""A value function based on a GaussianMLP model."""
+import torch
+from torch import nn
+
+from garage.torch.modules import GaussianMLPModule
+from garage.torch.value_functions.base import ValueFunction
+
+
+class GaussianMLPValueFunction(ValueFunction, GaussianMLPModule):
```

Comment:
Please don't inherit from `GaussianMLPModule`. We did this with policies, and it turned out to make it impossible to make the pytorch policies into a class hierarchy without some major hacks or undoing it. Instead, `ValueFunction` should inherit from `torch.nn.module`, and the `GaussianMLPModule` should just be a field.

Reply:
i agree.

Reply:
fixed